src/XMakeBuildEngine/UnitTests/Construction/ElementLocation_Tests.cs

```diff
@@ -112,7 +112,7 @@ public void TestLargeElementLocationUsedLargeColumn()
             }
             catch (InvalidProjectFileException ex)
             {
-                Assert.Equal(70012, ex.ColumnNumber);
+                Assert.Equal(1, ex.ColumnNumber);
```

Comment:
These results are *wrong*, though, aren't they? In `xplat` we have them disabled for netcore.